src/main/java/com/github/mauricioaniche/ck/metric/MethodLevelFieldUsageCount.java

```diff
@@ -46,34 +46,25 @@ public void endVisit(FieldAccess node) {
 		isFieldAccess = false;
 	}
 
-	private void addField(String var) {
+	private void plusOne(String var) {
 		if (!occurrences.containsKey(var))
 			occurrences.put(var, 0);
-	}
-
-	private void plusOne(String var) {
-		addField(var);
 		occurrences.put(var, occurrences.get(var) + 1);
 	}
 
 	public void visit(SimpleName node) {
+		String variableName = node.getIdentifier();
 
-		String var = node.getIdentifier();
-
-		if(isFieldAccess)
```

Comment:
Was this a bug?

Reply:
The call to addField was redundant because the plusOne function would do the same later. Thus, the function was also redundant once the redundant call was removed.